src/wp-includes/post.php

```diff
@@ -1641,15 +1641,15 @@ function get_post_types( $args = array(), $output = 'names', $operator = 'and' )
  *                                                         See get_post_type_capabilities().
  *     @type bool         $map_meta_cap                    Whether to use the internal default meta capability handling.
  *                                                         Default false.
- *     @type array        $supports                        Core feature(s) the post type supports. Serves as an alias for calling
+ *     @type array|bool   $supports                        Core feature(s) the post type supports. Serves as an alias for calling
```

Comment:
```suggestion
 *     @type array|false   $supports                        Core feature(s) the post type supports. Serves as an alias for calling
```

Let's make it explicit that only `false` is supported, not `true`